packages/modules/gcloud/src/spanner-emulator-container.test.ts

```diff
@@ -0,0 +1,57 @@
+import { getImage } from "../../../testcontainers/src/utils/test-helper";
+import { SpannerEmulatorContainer } from "./spanner-emulator-container";
+
+// select the fourth FROM in the Dockerfile (spanner emulator)
+const IMAGE = getImage(__dirname, 3);
+
+describe("SpannerEmulatorContainer", { timeout: 240_000 }, () => {
+  it("should start and expose endpoints", async () => {
+    // <example startup> {
+    const container = await new SpannerEmulatorContainer(IMAGE).withProjectId("test-project").start();
+    const grpcEndpoint = container.getEmulatorGrpcEndpoint();
+
+    expect(grpcEndpoint).toMatch(/localhost:\d+/);
```

Comment:
This test will fail depending on where it's run (host is not always `localhost`). Instead of testing the URL, is it possible to test the behaviour instead - i.e. that we're able to talk to use Spanner via GRPC?